app/routes.py

```diff
@@ -119,4 +119,21 @@ def settings():
         if credential_Form.newPassword.data != "":
             user.passwordHash = generate_password_hash(credential_Form.newPassword.data)
         dbSession.commit()
-    return render_template('settings.html', title="settings", credential_Form=credential_Form)
\ No newline at end of file
+    return render_template('settings.html', title="settings", credential_Form=credential_Form)
+
+@myapp_obj.route("/contacts")
+def view_contacts():
+    # create a new session to interact with the database
+    session = models.Session()
+
+    # get the user whose contact list we want to open
+    user_to_open_contact_list = session.query(models.user).filter_by(email='user@example.com').first()
```

Comment:
This shouldn't be and example email. There needs to be conditional logic display the current users contacts. Look into the session variable that we have stored. Not the database one but the flask session.